cmd/ctr/commands/images/tag.go

```diff
@@ -82,6 +90,9 @@ var tagCommand = cli.Command{
 		}
 		// Support multiple references for one command run
 		for _, targetRef := range context.Args()[1:] {
+			if _, err := reference.ParseAnyReference(targetRef); !context.Bool("skip-reference-check") && err != nil {
```

Comment:
```suggestion
			if !context.Bool("skip-reference-check") {
				if _, err := reference.ParseAnyReference(targetRef); err != nil {
```

Same suggestion as above.

Reply:
Done.